x-pack/plugins/osquery/public/live_queries/form/index.tsx

```diff
@@ -138,11 +145,15 @@ const LiveQueryFormComponent: React.FC<LiveQueryFormProps> = ({
 
   const onSubmit = useCallback(
     async (values: LiveQueryFormFields) => {
+      const dynamicQueryPresent = values.query && containsDynamicQuery(values.query);
       const serializedData = pickBy(
         {
           agentSelection: values.agentSelection,
           saved_query_id: values.savedQueryId,
-          query: values.query,
+          query:
```

Comment:
I am thinking.... whether we need to replace these data here? Why not only in create_action_handler, since we pass alertIds anyway and that ecsData down below?